src/layouts/components/Homepage/HeroHighlightSection.tsx

```diff
@@ -0,0 +1,195 @@
+                          <FormErrorMessage>
+                            {errors.highlights[highlightIndex].title}
+                          </FormErrorMessage>
+                        </FormControl>
+                        <FormControl
+                          isInvalid={
+                            !!errors.highlights[highlightIndex].description
+                          }
+                          isRequired
+                        >
+                          <FormLabel>Description</FormLabel>
+                          <Input
+                            placeholder="Highlight description"
+                            id={`highlight-${highlightIndex}-description`}
+                            value={highlightDescription}
+                            onChange={onChange}
+                          />
+                          <FormErrorMessage>
+                            {errors.highlights[highlightIndex].description}
+                          </FormErrorMessage>
+                        </FormControl>
+                        <FormControl
+                          isInvalid={!!errors.highlights[highlightIndex].url}
+                          isRequired
+                        >
+                          <FormLabel>URL</FormLabel>
+                          <Input
+                            placeholder="Insert /page-url or https://"
+                            id={`highlight-${highlightIndex}-url`}
+                            value={highlightUrl}
+                            onChange={onChange}
+                          />
+                          <FormErrorMessage>
+                            {errors.highlights[highlightIndex].url}
+                          </FormErrorMessage>
+                        </FormControl>
+                        <Button
+                          id={`highlight-${highlightIndex}-delete`}
+                          onClick={() =>
+                            onClick({
+                              target: {
+                                id: `highlight-${highlightIndex}-delete`,
+                              },
+                            })
+                          }
+                          alignSelf="center"
+                          variant="clear"
+                          colorScheme="critical"
+                          mt="1rem"
+                        >
+                          Delete option
+                        </Button>
+                      </Editable.Section>
+                    </Editable.DraggableAccordionItem>
+                  )
+                }
+              )}
+            </Editable.Section>
+          </Editable.EmptySection>
+        </Editable.Accordion>
+      </Editable.Droppable>
+      <Button
+        id={`highlight-${highlights.length}-create`}
+        onClick={onCreate}
+        variant="outline"
+        w="full"
+        leftIcon={<BiPlus fontSize="1.5rem" />}
+        isDisabled={highlights.length >= 4}
```

Comment:
nit: can extract to top of file as const MAX_HIGHLIGHTS = 4

Reply:
will do

Reply:
fixedzxc